### ISSUE TITLE
Crash on Fork Activation

### ISSUE THREAD

**Issue by noxthot** [ForkIssues/TrackerWin#995]
Hi there.

In our company we had the following reproducible error:
Clicking on **Help -> Fork Activation** crashes the application after a couple of seconds. No error message or dialog appears.

This happens with the newest stable version 1.54.2.0 (and I believe one developer even had a version < 1.50.0.0).

We also found a "fix" for this problem: Calling **File -> Check for Updates** before fixes the problem (even if there is no update available) as long as the application is up and running.

**Comment by DanPristupov** [comment]
The "fix" is really strange.
Do you work behind proxy/VPN on those machines?

**Comment by noxthot** [comment]
Yes - both is the case. Due to home office we are connecting to the company's network via Cisco VPN.
And within our company we are behind a proxy.

**Comment by DanPristupov** [comment]
We use paddle.com to handle the licensing. The process uses their own library and I'm not able to fix this.

Most likely what's going on is that the API request returns some VPN fallback response instead of JSON.

At the time Fork doesn't have online activation (https://github.com/fork-dev/TrackerWin/issues/790#issuecomment-631498526).

You can keep using Fork without activation, it will not stop working. I can also make a refund if you want.

**Comment by noxthot** [comment]
Too bad. Like I wrote we already found a work around. Just wanted the issue to be known. Thanks for your fast replies.

**Comment by dimizuno** [comment]
Hi. I have the same issue (also behind corporate Proxy+VPN). I can confirm that the above mentioned workaround to call "Ceck for Updates" first works for me as well.

**Comment by piercemar** [comment]
Hi,

I just bought a license and had the same issue (also behind corporate proxy).
The workaround worked for me too (checking for updates, then activating).
Could be an issue with proxy initialization/authentication (though no popup showed up).
